users/test.py

```diff
@@ -21,7 +21,7 @@
 
 import unittest
 import mongomock
-from app import find_authorization_in_db
+from app import find_authorization_in_db, upsert_user_in_db
```

Comment:
I'd have a general preference for the form:

```python
import app

# ...
app.find_authorization_in_db(...)
```

As this gives the reader a little bit more of a hint that the function has been imported from another module.

Reply:
Done https://github.com/knative-portability/large-events/pull/8/commits/14081bb4f73065e4ad403d07e73fc9e25e781250